testsuite/deployments/pom.xml

```diff
@@ -101,4 +143,5 @@
             </build>
         </profile>
     </profiles>
-</project>
+
+</project>
```

Comment:
Will you please add missing new line here?

Reply:
Fixed, thanks